bftengine/src/bftengine/MsgsCommunicator.cpp

```diff
@@ -60,8 +60,9 @@ void MsgsCommunicator::send(std::set<NodeNum> dests, char* message, size_t messa
 uint32_t MsgsCommunicator::numOfConnectedReplicas(uint32_t clusterSize) {
   uint32_t ret{0};
   for (uint32_t i = 0; i < clusterSize; ++i) {
-    if (communication_->getCurrentConnectionStatus(i) == ConnectionStatus::Disconnected) continue;
-    ++ret;
+    if (communication_->getCurrentConnectionStatus(i) == ConnectionStatus::Connected) {
+      ++ret;
+    }
```

Comment:
Hi Utkarsh - did you explicitly verify the count is correct after your change? What could help is for example a temporary  INFO log while running the Apollo tests, to make sure the number is correct?

Or maybe you validated using another method? If so, please provide more details in the **Testing done** section.

Thank you!
Teo